src/elm/Session/LoggedIn.elm

```diff
@@ -588,6 +624,116 @@ communitySelectorModal model =
         text ""
 
 
+addContactModal : Model -> Html Msg
+addContactModal ({ addContactInfo } as model) =
+    let
+        header =
+            div [ class "mt-4" ]
+                [ p [ class "inline bg-purple-100 text-white rounded-full py-0.5 px-2 text-caption" ]
+                    [ text "NEW" ]
+                , p [ class "text-heading font-bold mt-2" ]
+                    [ text "Add your Phone Number" ]
```

Comment:
The project has several languages, please, check `public/translations`. We add translations into related JSON files from that directory. For now, you can use Google Translate for those you don't speak. There are `t` and `tr` functions for translation strings from `src/elm/Session/Shared.elm:82`, you can find the examples all over the project.